buildspec.staging.yaml

```diff
@@ -6,21 +6,19 @@ phases:
       nodejs: 12
   build:
     commands:
-      - ls
       - npm install --unsafe-perm
       - npm run test
       - npm run buildProdBrowser
       - node -r esm integrationBuildScript.js
-      - sed 's|//# sourceMappingURL=rudder-analytics.min.js.map||' dist/rudder-analytics.min.js  > dist/prod.js
-      - mv dist/prod.js dist/rudder-analytics.min.js
+      - sed -i -e 's|rudder-analytics.min.js.map|rudder-analytics-staging.min.js.map|' -e 's|{{RS_BUGSNAG_API_KEY}}|'$RS_BUGSNAG_API_KEY'|' dist/rudder-analytics.min.js
       - aws s3 cp dist/rudder-analytics.min.js s3://$S3_BUCKET_NAME/v1.1/rudder-analytics-staging.min.js --cache-control max-age=3600 --acl public-read
+      - aws s3 cp dist/rudder-analytics.min.js.map s3://$S3_BUCKET_NAME/v1.1/rudder-analytics-staging.min.js.map --cache-control max-age=3600 --acl public-read
       - mkdir dist/temp
       - cp dist/integrations/* dist/temp
       - for file in dist/temp/* ; do old_name=$(basename "$file"); new_name=$(echo $old_name | awk '{split($0, tmp, "."); print sprintf("%s-staging.%s.%s", tmp[1], tmp[2], tmp[3])}'); mv "dist/temp/$old_name" "dist/temp/$new_name"; done
```

Comment:
We need to change the `soruceMappingURL` value inside the integration SDKs.

Reply:
Addressed